### ISSUE TITLE
|./multinode-demo/client.sh master.testnet.solana.com 50| crashes

### ISSUE THREAD

**Issue by mvines** [solana-labs/solana#678]
```
$ ./multinode-demo/client.sh master.testnet.solana.com 50
...
 INFO 2018-07-18T04:13:55Z: solana::nat: Using source address 35.199.175.144:9556 for gossip
 WARN 2018-07-18T04:13:55Z: solana::crdt: f8951108aa5468c2: LEADER_UPDATE TO 7fead0e765f85c03 from 0
 INFO 2018-07-18T04:13:55Z: solana::counter: COUNTER:{"name": "crdt-update-count", "counts": 100, "samples": 100,  "now": 1531887235160}
 WARN 2018-07-18T04:13:55Z: solana::crdt: f8951108aa5468c2: LEADER_UPDATE TO 470b128f548e80e7 from 7fead0e765f85c03
CONVERGED!
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `134`,
 right: `50`', src/bin/client-demo.rs:228:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:211
   3: std::panicking::default_hook
             at libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:463
   5: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:350
   6: solana_client_demo::main
             at src/bin/client-demo.rs:228
   7: std::rt::lang_start::{{closure}}
             at /checkout/src/libstd/rt.rs:74
   8: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:310
   9: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:105
  10: std::rt::lang_start_internal
             at libstd/panicking.rs:289
             at libstd/panic.rs:374
             at libstd/rt.rs:58
  11: std::rt::lang_start
             at /checkout/src/libstd/rt.rs:74
  12: main
  13: __libc_start_main
  14: _start
```

Interestingly, restarting the leader node with `sudo snap set solana mode=leader+drone` causes the LHS value to increase.

**Comment by aeyakovenko** [comment]
@mvines seems like you have old nodes
```
 WARN 2018-07-18T04:13:55Z: solana::crdt: f8951108aa5468c2: LEADER_UPDATE TO 470b128f548e80e7 from 7fead0e765f85c03
```
In theory you should only see an update from 0 -> hexnumber...

**Comment by mvines** [comment]
Fixed by #679
